tools/github/runStandaloneTests.sh

```diff
@@ -45,8 +45,8 @@ export KUBECONFIG="$(kind get kubeconfig-path)"
 kubectl config set-context --current --namespace=default
 
 # This is required because it is timed out to pull the image during the test.
-docker pull openwhisk/action-nodejs-v14:nightly
-docker pull openwhisk/dockerskeleton:nightly
+docker pull openwhisk/action-nodejs-v14:1.20.0
+docker pull openwhisk/dockerskeleton:1.15.0
 docker pull openwhisk/example:nightly
```

Comment:
One thing I am uncertain about is this image(`openwhisk/example`).
I couldn't find where this image comes from yet.
The latest non-hash-based image tag is [1.0.0](https://hub.docker.com/layers/openwhisk/example/1.0.0/images/sha256-9f9b07a1e7477adf861dd8eca2fc0203250fb3672668445a684b9b40fe424e7d?context=explore) but it has been 6 years, so I am not sure that is what we want.

@dgrove-oss Do you have any idea?

Reply:
The openwhisk/example image comes from openwhisk-runtime-docker.  I think we lost track of the need to use multiple tags when we do a release of openwhisk-runtime-docker to cause this image to be tagged as well.  I just pushed the missing tag.  If that doesn't work, I can update manually.   It will end up being the same tag as the dockerskeleton (1.15.0)

Reply:
@dgrove-oss Thank you so much. I've updated it.
[It's available now](https://hub.docker.com/layers/openwhisk/example/1.15.0/images/sha256-046a02256e20c164624ff74a64c9289c61d0f4d922eab587a26480d2acb61753?context=explore).